### PR TITLE
Popularity

### PR DESCRIPTION
**Pull request by rs-randallburt**

### CODE REVIEW

File: rackspace.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@ application-name: Drupal
 application-version: 7.32
 flavor: Multiple Linux servers
 flavor-weight: 1
+popularity: 5
```
Comment (adampickeral) [inline rackspace.yaml]: question on this: how is this template in the top 5 when it's not even returned in the template catalog? drupal-single is the only drupal template in the catalog

Comment (rillip3) [inline rackspace.yaml]: Drupal-multi was disabled by Jason Boyles a while back as he was doing a rework on it. I imagine this ranking might have come from already created stacks.

Comment (rs-randallburt) [inline rackspace.yaml]: The popularity rank is for the application, not the individual templates, so the rank is based on the single server version as well.

Comment (adampickeral) [inline rackspace.yaml]: oh that's right, application, not single template. :+1:

**Pull request merged**